content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/detect-log4shell-demo/index.md

```diff
@@ -258,27 +270,137 @@ data:
   log4shell-demo-policy: ...
 ```
 
-You can find all records in the cluster containing SBOMs indicating the Log4Shell vulnerability using the following command.
+After confirming that the policy is active, try creating a Kubernetes Pod using the vulnerable image again.
 
 ```sh
-chainctl cluster records ls $CLUSTER
+kubectl run log4shell-demo --image=ghcr.io/chainguard-dev/log4shell-demo/app:v0.1.0
 ```
 
-Under the `POLICIES` column you'll receive output that indicates the policy has failed.
+This will again result in an error.
 
 ```
-                                       IMAGE                                      |            POLICIES            |   WORKLOADS   | PACKAGES | LAST SEEN | LAST REFRESHED  
-----------------------------------------------------------------------------------+--------------------------------+---------------+----------+-----------+-----------------
-  ghcr.io/chainguard-dev/log4shell-demo/app@sha256:ba4037…                        | log4shell-demo-policy:fail:40s | Pod:1         | maven:2  | 3m49s     | sbom:39s        
+Error from server (BadRequest): admission webhook "enforcer.chainguard.dev" denied the request: validation failed: failed policy: log4shell-demo-policy: spec.containers[0].image
+ghcr.io/chainguard-dev/log4shell-demo/app@sha256:ba4037061b76ad8f306dd9e442877236015747ec42141caf504dc0df4d10708d no matching attestations with type cyclonedx
+```
+
+However, this error message is different from the previous one. This time the error mentions that the Log4Shell image doesn't have a CycloneDX attestation. This means that Chainguard Enforce successfully applied the `log4shell-demo-policy` and used it to block the image from running.
```

Comment:
I mentioned that this draft isn't perfect - this is what I was referring to. Ideally, this would block the container from running b/c it contains a vulnerable version of Log4J. Will update this down the line when I have an image/policy that works to my liking.